test.js

```diff
@@ -900,8 +872,7 @@ test('createAsyncIterator creates AsyncIterator from another AsyncIterator', ()
 })
 
 test('createAsyncIterator creates AsyncIterator for string literal', () => {
-  // $FlowFixMe expected function override to apply.
-  var iterator = createAsyncIterator('ABC')
+  var iterator = createAsyncIterator((('ABC': any): {length: number}))
```

Comment:
Very interesting! Iterating over string is weird edge case anyhow. I can account for that